core/lib/config/src/configs/experimental.rs

```diff
@@ -60,3 +61,37 @@ impl ExperimentalDBConfig {
         100
     }
 }
+
+#[derive(Debug, Deserialize, Clone, PartialEq)]
+pub struct ExperimentalVmPlaygroundConfig {
+    /// Mode in which to run the fast VM implementation. Note that for it to actually be used, L1 batches should have a recent version.
+    #[serde(default)]
+    pub fast_vm_mode: FastVmMode,
+    /// Path to the RocksDB cache directory.
+    #[serde(default = "ExperimentalVmPlaygroundConfig::default_db_path")]
```

Comment:
Note: `serde(default)` doesn't work with file-based config which we already use on stage.

Reply:
I've [accounted for defaults](https://github.com/matter-labs/zksync-era/blob/83156f08c2c01e282a5eb5d5be6cfcf8f62cc6aa/core/lib/protobuf_config/src/experimental.rs#L75) in `protobuf_config` crate, which AFAIU is sufficient to make file-based config work.